### DISCUSSION TITLE
Fresh install from homepage

### DISCUSSION THREAD

**Discussion by madmaiike** [tteck/Proxmox#976] [Help Needed] [answered]
Hi there, 

Just recently got back to building a new homeassistant server moved from RPi to NUC (day and night) and your scripts are awesome. I even discovered some apps didn't know existed, like homepage. 

I've ran the script and the LXC is running except I cannot access it through http://ipadress:3000 and when I console into it I'm facing a login prompt. I have no idea of user and pass - did I miss it in the manual ? 

Thanks 

Michael

**Comment by tteck** [comment]
If it's asking for credentials in the LXC console, and you didn't set a password during creation (Advanced) you goofed the install. Probably by moving out of the shell. Remove and start over.

**Reply by tteck** [reply]
Wait until you see `✓ Completed Successfully!` before leaving the shell

**Reply by madmaiike** [reply]
Exactly my thoughts I goofed something because these scripts are fire 🔥 

Doing it over now using advanced and setting a root password. 

Was about to delete the question that was quick ! 

Thanks.

**Reply by tteck** [reply]
No matter which settings you use (Default or Advanced), wait for `✓ Completed Successfully!`

**Reply by tteck** [reply]
It may have been a bug I just fixed.

**Reply by madmaiike** [reply]
I will try again and update you - was late already.

**Comment by tteck** [comment, marked as answer]
Script has been updated to resolve issue

**Comment by madmaiike** [comment]
Confirmed - Now it works :) 

I've discovered Homepage thanks to you - can't wait to customise it. 

Thanks - great scripts.
